InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using FluentAssertions;
+using Xunit;
+
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
+{
+    public partial class HomeApiTests
+    {
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            string expectedMessage = "Thank you, Mario. But the princess is in another castle!";
```

Comment:
All tests should be roken into `// given`, `// when` and `// then` just like it shows in the examples, videos and existing code in other projects

Reply:
Okay . On it 👍🏻